scripts/firedrake-install

```diff
@@ -980,39 +978,35 @@ def build_and_install_libspatialindex():
 
 def build_and_install_slepc():
     petsc_dir, petsc_arch = get_petsc_dir()
-
+    slepc_dir, slepc_arch = get_slepc_dir()
+    assert petsc_arch == slepc_arch
```

Comment:
Is it obvious that these should match?

Reply:
Well, the definition of `get_slepc_dir` just returns `petsc_arch` in the second slot (SLEPc doesn't have a separate idea of the arch, it just matches what petsc uses). This is belt-and-braces assertion.